lib/graphiti/debugger.rb

```diff
@@ -187,8 +187,8 @@ def flush_chunk(chunk, depth = 0)
     end
 
     ActiveSupport::Notifications.subscribe \
-      "graphiti.data", method(:on_data)
+      "data.graphiti", method(:on_data)
```

Comment:
Should change `data` out for `resolve` here.

Reply:
Fixed.